tests/playbooks/tests_bond.yml

```diff
@@ -8,6 +13,8 @@
     dhcp_interface1: test1
     port2_profile: bond0.1
     dhcp_interface2: test2
+  tags:
+    - "tests::expfail"
```

Comment:
Why is this expected to fail? Would be good to explain/link to an issue/BZ to track this.